encoder/vaapiencoder_h264.cpp

```diff
@@ -1014,6 +1194,10 @@ YamiStatus VaapiEncoderH264::reorder(const SurfacePtr& surface, uint64_t timeSta
     }
 
     picture->m_poc = m_frameIndex * 2;
+    picture->m_priorityId = m_videoParamAVC.priorityId;
+    picture->m_temporalId
+        = TempIds[m_temporalLayerNum - 1][m_frameIndex % H264_MIN_TEMPORAL_GOP];
```

Comment:
Code will more cleanly if we hide this detail in an inline function. Like `inline uint32_t getTemporaId(int frameIndex)`.